### PR TITLE
Upload and display image

### PR DESCRIPTION
**Pull request by lyrated**
upload image using "multer" package.
image is not saved per sé, but the string to display the data is saved in mongodb.... not a clean solution i guess, but should be enough for us :P
FOR NOW you can't edit your app image. if we have time and motivation we can add that feature later~
grafik

### CODE REVIEW

**Comment by kimc0de** [comment]
It wasn't successfully added for me...and i got this weird error

https://user-images.githubusercontent.com/61560491/123093951-977af780-d42c-11eb-8435-bfcd681166bc.mov

**Comment by lyrated** [comment]
@kimc0de can you try creating the folder "uploads" inside images and try again? I added this folder to .gitignore but I'll add the empty folder there if that's what messing it up

**Comment by kimc0de** [comment]
@lyrated  ok it works with the uploads folder.. if you add to git ignore it won't be push to main right? then production will fail?

overall looks good to me. i'll see what i can do for the input style lol..

File: views/greenPickApp/newApp.ejs [hunk 1]
```diff
@@ -50,14 +51,19 @@
...
-             data-max-file-size="3MB"
-             data-max-files="1">
+    <% if (appImageExists) { %>
+      <h3 class="h4">App logo:</h3>
+      <div class="background">
+        <img src="<%= app.image %>" class="app-logo" alt="Your app logo for <%= app.name %>">
+      </div>
+    <% } else { %>
+      <label class="question-field" for="upload-image">App logo:</label>
+      <input type="file" name="image"id="upload-image" accept="image/*" required="">
```
Comment (kimc0de) [inline views/greenPickApp/newApp.ejs:61]:
```suggestion
      <input type="file" name="image" id="upload-image" accept="image/*" required="">
```

**Comment by kimc0de** [comment]
@lyrated i just push the style, it works the same. you can check again and merge. :D

**Pull request merged**